less/admin/UsersListPage.less

```diff
@@ -0,0 +1,34 @@
+.UsersListPage {
```

Comment:
I'm not sure how well this will scale on mobile if new columns are added

Reply:
Now it should scroll horizontally when the viewport is shrinked.